### PR TITLE
takealot_importer: pull various additions

### PR DESCRIPTION
**Pull request by madmouser1**
- 1. Added link to high res image
- 2. Added import warning if information is buggy
- 3. Parse additional information from Product Information Tab (Release title and label)
- 4. Changed logic to parse different formats in Description tab
    - < br > based formatting
    - < div > based formatting
    - console log and release_maybe_buggy set if unknown

### CODE REVIEW

File: takealot_importer.user.js [hunk 1]
```diff
@@ -629,4 +635,4 @@ Countries["Heard Island and McDonald Islands"] = "HM";
 Countries["Iran, Islamic Republic of"] = "IR";
 Countries["Saint Pierre and Miquelon"] = "PM";
 Countries["Saint Helena"] = "SH";
-Countries["Svalbard and Jan Mayen"] = "SJ";
+Countries["Svalbard and Jan Mayen"] = "SJ";
```
Comment (zas) [inline takealot_importer.user.js]: This removing of trailing newline is unrelated to the change. Nothing serious, but it should be avoided.

File: takealot_importer.user.js [hunk 2]
```diff
@@ -61,21 +63,289 @@ function insertMbUI(mbUI) {
...
 // Insert links to high res image in Takealot page
 function insertIMGlinks() {
 	var imghref = $('#slideshow a.jqzoom').attr('href');
 	LOGGER.debug("insertIMGlink Firing", imghref);
 	$('#slideshow').append('<p><img src="//musicbrainz.org/favicon.ico" /><a href="' + imghref + '">MB High Res Image</a></p>');
 }
 
+
+// Artist in product information is without diacritics
+// Remove Diacritics from Artist in header to comapre with product information
```
Comment (zas) [inline takealot_importer.user.js]: s/comapre/compare/

**Comment by zas** [comment]
Overall, looks ok to me, if you agree i'll merge it (if you don't have any commit to add).

**Comment by madmouser1** [comment]
Many thanks, no more commits, will be bug fixes and new formats as they are uncovered.

**Pull request merged**